src/integrations/GA4/browser.js

```diff
@@ -24,7 +24,6 @@ export default class GA4 {
     this.sendUserId = config.sendUserId || false;
     this.blockPageView = config.blockPageViewEvent || false;
     this.extendPageViewParams = config.extendPageViewParams || false;
-    this.extendGroupPayload = config.extendGroupPayload || false;
```

Comment:
we are going to communicate this to the customer right? Because it will be a breaking change for those who have opted to not send traits even though they are sending it in payload

Reply:
Yes, we have prepared doc for new fixes. CS team will communicate with the customers regarding fixes